pkgs/by-name/sv/svndumpsanitizer/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, gccStdenv
+, fetchFromGitHub
+}:
+
+gccStdenv.mkDerivation {
```

Comment:
Just a question: is there a reason behind the choice of `gccStdenv` over `stdenv`?

Reply:
I opted for gccStdenv in the initial, upstream Flake just because the upstream readme explicitly called out gcc in its build instructions. I don't have any reason to suspect that clang would _not_ work, I just haven't tested any of the resulting binary's actual functionality.

Reply:
Maybe you could put it back to `stdenv`, set `meta.platforms = platforms.unix`, and let ofborg have a go at building it?

Reply:
Thanks for the suggestion! I've given it a try.